README.md

```diff
@@ -139,19 +139,21 @@ You can remove cached records by their ID.
 
 ```console
 $ jcache cache list
-  ID  URI                                    Created           Accessed
+  ID  Origin URI                             Created           Accessed
 ----  -------------------------------------  ----------------  ----------------
-   5  tests/notebooks/external_output.ipynb  2020-02-29 17:48  2020-02-29 17:48
-   4  tests/notebooks/complex_outputs.ipynb  2020-02-29 17:48  2020-02-29 17:48
-   3  tests/notebooks/basic_unrun.ipynb      2020-02-29 17:48  2020-02-29 17:48
-   2  tests/notebooks/basic_failing.ipynb    2020-02-29 17:48  2020-02-29 17:48
+   5  tests/notebooks/external_output.ipynb  2020-03-12 17:31  2020-03-12 17:31
+   4  tests/notebooks/complex_outputs.ipynb  2020-03-12 17:31  2020-03-12 17:31
+   3  tests/notebooks/basic_unrun.ipynb      2020-03-12 17:31  2020-03-12 17:31
+   2  tests/notebooks/basic_failing.ipynb    2020-03-12 17:31  2020-03-12 17:31
 ```
 
+Tip: Use the `--latest-only` option, to only show the latest versions of cached notebooks.
```

Comment:
What do you think about the default being `--latest-only` and having a flag like `--include-history` instead?

Reply:
The reason why I'm hesitant to use `--include-history` is that the terminology is not strictly *true* in how the cache actually operates. Filtering for a certain `Origin URI` and sorting by `Created` **does not** guarantee to show a history for that URI; for example if you start changing file names of notebooks, or for a specific example:

- create a git repository with `notebook.ipynb` and cache it
- switch the branch of the git repository, modify `notebook.ipynb` and cache that
- switch back to the original git branch

Assuming the actual cache isn't committed to git, you now have two versions of `notebook.ipynb` in the cache and the **older** one is actually the latest for the branch you are currently on.

Reply:
#34 could be an alternative way to show only particular records

Reply:
Yeah, that makes sense. I am just trying to think of the most common use-case, and generally speaking we should be tailoring the default UI around the common use-cases. I think it's much more common for someone to think "I want to see a list of the notebooks that are in the cache" than for them to also think "and I want to see  values for every version that I have ever cached for a given notebook".

Though your branching example is a tricky one, that would definitely confuse people :-/

Reply:
I’m definitely open to some form of solution, but it should be carefully considered what the balance is between ease-of-use and compliance with underlying logic